src/styles/blocks/catalog.scss

```diff
@@ -18,6 +18,13 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition-duration: $animationDuration;
```

Comment:
Add `transition-property` properties too
Without it, transition applies to all the properties which is not good for performance
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x

Reply:
Thank you
Done)